tests/functional/api/groups/members_test.py

```diff
@@ -116,7 +22,6 @@ def test_it_returns_list_of_members_for_restricted_group_without_auth(
 
         res = app.get(
             "/api/groups/{pubid}/members".format(pubid=group.pubid),
-            params={"page[offset]": 1, "page[limit]": 2},
```

Comment:
I removed the pagination query params from these functests, so they're now testing that the default behaviour of the API (without these query params) is paginated. Added a separate test for pagination below.